frontend/app/services/MasterclassDataExtractor.scala

```diff
@@ -15,7 +15,7 @@ object MasterclassDataExtractor {
 
   def extractEventbriteInformation(content: Content): Seq[MasterclassData] = {
     val eventbriteIdsFromRefs =
-      content.references.filter(_.`type` == "eventbrite").map(_.id.stripPrefix("eventbrite/"))
+      content.references.toSeq.filter(_.`type` == "eventbrite").map(_.id.stripPrefix("eventbrite/"))
 
     val eventbriteIds = (eventbriteIdsFromRefs ++ scrapeEventbriteIdsFrom(content)).distinct
 
```

Comment:
the MembersService diff below is collapsed due to being "large" but actually it's An important change, I deleted most of the code as it was unused in the end.